src/NzbDrone.Core/Download/Clients/Porla/Models/PorlaTorrent.cs

```diff
@@ -0,0 +1,36 @@
+using Newtonsoft.Json;
+using NzbDrone.Core.Download.Clients.LibTorrent.Models;
+
+namespace NzbDrone.Core.Download.Clients.Porla.Models
+{
+    /// <summary> Wraps the LibTorrent Infohash type into a Porla Type for easier handling </summary>
+    public sealed class PorlaTorrent
+    {
+        [JsonProperty("info_hash", NullValueHandling = NullValueHandling.Ignore)]
+        [JsonConverter(typeof(LibTorrentInfoHashConverter))]
+        public LibTorrentInfoHash InfoHash { get; set; }
+
+        [JsonConstructor]
+        public PorlaTorrent(string hash, string status)
+        {
+            InfoHash = new LibTorrentInfoHash(hash, status);
+        }
+
+        public PorlaTorrent(LibTorrentInfoHash ltif)
```

Comment:
Use words instead of acronyms

```suggestion
        public PorlaTorrent(LibTorrentInfoHash infoHash)
```